doc/performance-investigations/0005-replacement-of-flask-caching/summary.md

```diff
@@ -0,0 +1,78 @@
+# Replacement of Flask Caching with Built-in functools caching
+
+An improvement in Datastore latencies was seen coinciding with the replacement of Flask caching with Python 3.8 built-in functools caching. 
+This test was carried out to help understand if these gains were coincidental or whether there is potentially some bottlenecks to the way we initialise the Datastore client.
+
+The tests documented here are like-for-like with our daily performance tests.
+
+
+## Benchmark version
+
+| Tag | Commit |
+|--------|-------|
+| latest | 8f2b9fe14eb19e19140f18b0e5bee78f4dbcb1af |
+
+## Benchmark profile
+
+| Option | Value |
+|--------|-------|
+| Requests file | census_household_gb_eng.json |
+| Run time | 60 mins |
+| Instances | 1 |
+| User wait time minimum | 1 |
+| User wait time maximum | 2 |
+| Clients | 64 |
+| Hatch rate | 64 |
+
+## Runner images
+| Type      | PR    | Commit |  Date | Comments |
+|-----------|--------|--------|--------|--------|
+| Baseline  | https://github.com/ONSdigital/eq-questionnaire-runner/pull/140  | cdf61f7290b62e97a901ed5bbb1396700d2c4494 | Tue 09, Jun 2020 10:15 UTC | Commit in which Flask caching was removed |
+| Investigation Candidate | https://github.com/ONSdigital/eq-questionnaire-runner/pull/141 | d4cdb5040dc0fc77270038b3af6e17a205ea077e | Mon 08, Jun 2020 07:52 UTC | Last commit before Flask caching/werkzeug cached_property was removed |
```

Comment:
What does the PR link relate to?

I think it would be clearer if we just made the commit sha link to the commit.

Reply:
Not sure why I added the PR. Adjusted.